core/src/main/java/hudson/model/AsyncAperiodicWork.java

```diff
@@ -80,8 +127,8 @@ public void run() {
                         l.closeQuietly();
                     }
 
-                    logger.log(Level.INFO, "Finished "+name+". "+
-                        (System.currentTimeMillis()-startTime)+" ms");
+                    logger.log(getNormalLoggingLevel(), "Finished {0}. {1,number} ms",
+                            new Object[]{name, System.currentTimeMillis() - startTime});
```

Comment:
NIT: the time/duration in the log will be different to that in the tasklog.
(not a new issue)